data/caasp/update.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -ex
+
+# 2.0 -> 2.0 http://download.suse.de/ibs/Devel:/CASP:/2.0:/ControllerNode:/TestUpdate/standard/Devel:CASP:2.0:ControllerNode:TestUpdate.repo
+# 
+
+while getopts "r:" opt; do
+case $opt in
+	r)
+		REPO=$OPTARG
+		;;
+	:)
+		echo "Option -$opt requires an argument." >&2
+		;;
+esac
+done
+
+saltid=$(docker ps | grep salt-master | awk '{print $1}')
+where="-P roles:(admin|kube-(master|minion))"
+srun="docker exec -i $saltid salt --batch 7"
+runner="$srun $where cmd.run"
+
+# Remove non-existent ISO repository
+# $runner "zypper rr 1"
+
+# Add repository as system venodors
+$runner 'echo -e "[main]\nvendors = suse,opensuse,obs://build.suse.de,obs://build.opensuse.org" > /etc/zypp/vendors.d/vendors.conf'
+$runner "zypper ar --refresh --no-gpgcheck $REPO"
+
```

Comment:
After the addition of the repositories, I would like to see what repositories are added. It will make our debugging life easier: `$runner zypper lr -U` should suffice